src/firefly/java/edu/caltech/ipac/firefly/server/query/ObsCorePackager.java

```diff
@@ -56,22 +55,29 @@ private List<FileGroup> computeFileGroup(DownloadRequest request) throws DataAcc
         String productTypes = request.getParam(PRODUCTS); //products to download in datalink file
         String[] products = (productTypes != null && !productTypes.trim().isEmpty()) ? productTypes.split(",") : null;
 
+        //todo: these are only to be used for cutout service descriptor - if missing, fallback and use ra/dec in service descriptor if available
+        String centerColParam = request.getParam("centerCols");
+        String[] centerCols = (centerColParam != null && !centerColParam.trim().isEmpty()) ? centerColParam.split(",") : null;
+        String cutoutValue = request.getParam("cutoutValue");
+
         List<FileInfo> fileInfos = new ArrayList<>();
         MappedData dgDataUrl = EmbeddedDbUtil.getSelectedMappedData(request.getSearchRequest(), selectedRows); //returns all columns
 
         try {
             Map<String,Integer> dataLinkFolders = new HashMap<>();
             for (int idx : selectedRows) {
                 String access_url = (String) dgDataUrl.get(idx, "access_url");
+
+                //todo: check that centerCols is not null before accessing the values - these will be used for cutouts
+                String ra =  Double.toString((Double) dgDataUrl.get(idx, centerCols[0]));
+                String dec = Double.toString((Double) dgDataUrl.get(idx, centerCols[1]));
+
                 if (access_url == null) {
                     continue; //no file available to process
                 }
                 String access_format = (String) dgDataUrl.get(idx, "access_format");
                 URL url = new URL(access_url);
 
-                FileInfo fileInf1o1 = new FileInfo(access_url, null, 0);
-                FileInfo fileInf1o2 = new FileInfo(access_url, ".txt", 0);
-
                 String colNames = request.getSearchRequest().getParam("templateColNames");
                 boolean useSourceUrlFileName= request.getSearchRequest().getBooleanParam("useSourceUrlFileName",false);
```

Comment:
In my opinion, the logic for resolving `ext_file_name` is incorrect.
Unless we know what the filename should be, we should defer naming it until the file is actually downloaded. At that point, the `Content-Disposition` header can be inspected to determine the proper filename.